protocols/ping/src/handler.rs

```diff
@@ -295,8 +295,7 @@ impl ConnectionHandler for Handler {
                         break;
                     }
                     Poll::Ready(Ok((stream, rtt))) => {
-                        log::debug!("latency to {} is {}ms", self.peer, rtt.as_millis());
-
+                        tracing::event!(tracing::Level::TRACE, peer = %self.peer, ms = rtt.as_millis());
```

Comment:
`Duration` actually has a very nice looking `Debug` impl so I'd suggest we use that:

```suggestion
                        tracing::event!(tracing::Level::TRACE, peer = %self.peer, ?rtt);
```